qa/run_xfstests.sh

```diff
@@ -46,7 +51,7 @@ TESTS="-g auto"	# The "auto" group is supposed to be "known good"
 
 # rbd presents geometry information that causes mkfs.xfs to
 # issue a warning.  This option avoids this class of problems.
-XFS_MKFS_OPTIONS="-l su=32k"
+#XFS_MKFS_OPTIONS="-l su=32k"
```

Comment:
Actually, this option is obsolete with recent versions of mkfs.xfs (in xfsprogs), which autodetects the correct stripe unit size. I left the comments in case this is ever run (either accidentally or intentionally) with xfsprogs from a distribution. Want me to remove them?